extension/index.googleFunction.js

```diff
@@ -8,6 +8,17 @@ const logger = utils.getLogger()
 exports.extensionTrigger = async (request, response) => {
   try {
     const authToken = auth.getAuthorizationRequestHeader(request)
+    if (!request?.body?.resource) {
+      response.status(400).send({
```

Comment:
Use return everywhere and don't omit it. Otherwise you could have weird behaviour like here where you don't use return - it will send 400, but the code will continue.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/630/commits/8b9884d0f3fd366b1ec5feb1d318f7a05432ecd5